airbyte-scheduler/src/main/java/io/airbyte/scheduler/SchedulerApp.java

```diff
@@ -94,12 +95,15 @@ public void start() throws IOException {
     final JobScheduler jobScheduler = new JobScheduler(jobPersistence, configRepository);
     final JobSubmitter jobSubmitter = new JobSubmitter(workerThreadPool, jobPersistence, configRepository, workerRunFactory);
 
+    Map<String, String> mdc = MDC.getCopyOfContextMap();
```

Comment:
I assume this change to enable logging in threads?

I am surprised that you have to do anything for the threads to log properly. If it doesn't work out of the box, it means that our log4j configuration file isn't properly configured.

Reply:
How did you decide it was the right solution to fix the pb?